Latin Abbreviations "e.g." Updated To "for example" (#19182)
kubernetes/website: content/en/docs/tasks/administer-cluster/sysctl-cluster.md

scheduled, but will fail to launch.

With the warning above in mind, the cluster admin can allow certain _unsafe_

> sysctls for very special situations like for example high-performance or real-time

Reviewer: ```suggestion
sysctls for very special situations such as high-performance or real-time
```

application tuning. _Unsafe_ sysctls are enabled on a node-by-node basis with a

> flag of the kubelet, e.g.:

Reviewer: ```suggestion
flag of the kubelet; for example:
```
